tests/integration/test_staff_tenant.py

```diff
@@ -37,9 +37,9 @@ class TestStaffTenant:
     def test_when_provided_no_staff_params(
         self, valid_header, create_tenant, create_join_staff
     ):
-        tenant_1 = create_tenant()
-        tenant_2 = create_tenant()
-        keep_tenant = create_tenant()
+        tenant_1 = create_tenant([])
```

Comment:
I'm not fond of having to know that an empty array needs to be passed in to the fixture in order for the method call to work. Can we make the parameter optional?